### ISSUE TITLE
FAILURE: Build failed with an exception.

### ISSUE THREAD

**Issue by tuanha2000vn** [vrtdev/flutter_workmanager#138]
```
Launching lib\main.dart on Android SDK built for x86 in debug mode...
D:\MyDocuments\Flutter\flutter_workmanager\example\android\app\src\main\AndroidManifest.xml:5:5-25:19 Error:
	tools:replace specified at line:5 for attribute android:name, but no new value specified
D:\MyDocuments\Flutter\flutter_workmanager\example\android\app\src\main\AndroidManifest.xml Error:
	Validation failed, exiting
```
```
[√] Flutter (Channel beta, v1.13.6, on Microsoft Windows [Version 10.0.18362.592], locale vi-VN)
    • Flutter version 1.13.6 at C:\flutter_stable\flutter
    • Framework revision 659dc8129d (3 weeks ago), 2019-12-30 09:24:47 -0800
    • Engine revision bdc9708d23
    • Dart version 2.8.0 (build 2.8.0-dev.0.0 c547f5d933)
```

**Comment by mr-** [comment]
Hey, I am not sure how the these Manifests are merged, but replacing 
```tools:replace="android:name"```
by 
```android:name="io.flutter.app.FlutterApplication"```
works for me.

**Comment by petermnt** [comment]
Fixed in #144
